### ISSUE TITLE
Can I get the topology of SD Link?

### ISSUE THREAD

**Issue by August-en** [jike5/P-MapNet#11]
How can I obtain the topology of SD Link?

For example, Link_A -> Link_B -> Link_C, how can I get the information which Link_B's incoming is Link_A, and Link_B's outgoing is Link_C?

**Comment by jike5** [comment]
> How can I obtain the topology of SD Link?
> 
> For example, Link_A -> Link_B -> Link_C, how can I get the information which Link_B's incoming is Link_A, and Link_B's outgoing is Link_C?

yes, you can definitely get it, maybe this repo(https://osmnx.readthedocs.io/en/stable/getting-started.html) will help.
